### DISCUSSION TITLE
How can i disable gin RedirectTrailingSlash?

### DISCUSSION THREAD

**Discussion by kadirgun** [goravel/goravel#499] [Q&A]
https://github.com/gin-gonic/gin/blob/9d7c0e9e1a301f417df9dc89a8cadc3bf9063db2/gin.go#L97-L102

When I try to access a /user/ route as /user, it redirects with gin 307. This causes a CORS error.

**Comment by hwbrzzl** [comment]
What's your real route? `/user/` or `/user`?

**Reply by kadirgun** [reply]
My real route is /user/

**Reply by krishankumar01** [reply]
There is an issue for a similar problem https://github.com/goravel/goravel/issues/408. Welcome to PR!
